rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -1995,8 +1996,40 @@ message ClaimSwapNeuronsRequest {
     repeated NeuronId followees = 8;
   }
 
+  message NeuronsFund {
```

Comment:
I'm confused by the name of this... it's not actually a NeuronsFund is it?  Isn't that something that exists in the NNS?

And the fields in it don't seem to describe a neurons fund, but an individual neuron?

Reply:
You're right, maybe it's a bad name, it is the information that is needed for a Neurons' Fund participant in a ClaimSwapNeuronsRequest